ui/src/app/cohort-search/modifier-page/modifier-page.component.tsx

```diff
@@ -349,10 +346,8 @@ export const ModifierPage = fp.flow(withCurrentWorkspace(), withCurrentCohortSea
       this.setState({count: null, formState}, () => this.validateValues());
     }
 
-    updateMods() {
-      const {cohortContext} = this.props;
-      const {formState} = this.state;
-      cohortContext.item.modifiers = formState.reduce((acc, mod) => {
+    getModifiersFromForm() {
```

Comment:
Moved this to a separate function since I noticed the calculate function was not using the current values in the form, usually resulting in an empty array getting sent with the request.